server/channels/db/migrations/postgres/000129_scheduled_posts_indexes.up.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX IF NOT EXISTS idx_scheduledposts_userid_channel_id_scheduled_at ON ScheduledPosts (UserId, ChannelId, ScheduledAt DESC);
```

Comment:
Implying sort order on a multi-column index ties it to the query very closely. If you are 100% sure on this, then go ahead. But I'd also suggest to add a unit test because now the query performance will be very sensitive to ordering changes.

Reply:
I removed the sort order as I'll may have to sort it differently later on.